src/Pelagos/Bundle/AppBundle/Rabbit/Consumer/DoiConsumer.php

```diff
@@ -146,7 +146,7 @@ protected function issueDoi(Dataset $dataset, array $loggingContext)
             try {
                 $doiUtil->createDOI(
                     $generatedDOI,
-                    $this->generateUrl('pelagos_homepage') . '/tombstone/' . $dataset->getUdi(),
+                    'https://data.gulfresearchinitiative.org/tombstone/' . $dataset->getUdi(),
```

Comment:
Yes, this is more data than a link. So in this case it's fine that it's hard coded.